openstack/networking/v2/extensions/layer3/floatingips/requests.go

```diff
@@ -124,6 +124,7 @@ type UpdateOptsBuilder interface {
 type UpdateOpts struct {
 	Description *string `json:"description,omitempty"`
 	PortID      *string `json:"port_id,omitempty"`
+	FixedIP     string  `json:"fixed_ip_address,omitempty"`
```

Comment:
`FixedIPAddress` - the field should always match the JSON tag.

Reply:
I took it from [this](https://github.com/gophercloud/gophercloud/blob/df38e1611dbe76a187bbcec07797714af9671066/openstack/networking/v2/extensions/layer3/floatingips/requests.go#L18) place

Reply:
Ah - good catch. Looks like this package a few other tag inconsistencies.

Let's keep it at `FixedIP` then. I'll merge this in the morning. Thanks!